src/components/LogIn/LogIn.tsx

```diff
@@ -71,6 +73,15 @@ export default function LogIn() {
     });
   }, [visitURLs]);
 
+  useEffect(() => {
+    if (userIsLoggedIn) {
+      sendAnalytics({
+        category: "Authentication",
+        action: "User Login",
```

Comment:
It might be helpful to also include the type of authentication:

```suggestion
        action: `User Login (${config?.authMethod})`,
```

Reply:
Done